db/background_mgr_resync_dcp.go

```diff
@@ -65,6 +75,16 @@ func (r *ResyncManagerDCP) Init(ctx context.Context, options map[string]interfac
 		return nil
 	}
 
+	// Get collectionIds and store in manager for use in DCP client later
```

Comment:
I think we might only want to set this in `newRunInit`. I don't think this will change from run to run but we don't need to do this calculation.

Reply:
It shouldn't change from run to run but it could (currently we don't handle that). I filed CBG-4473 for this last week.

My thinking of putting this here is for when CBG-4473 is done it will be easy to find a difference between the passed in collections through the endpoint and the previous interrupted run. In this case we should fail and force the user to reset resync. 
Example:
- Run resync with collection A 
- Stop resync 
- Call action=start again it with payload of Collection A and B 
- Here we should fail and force user to reset but we don't currently